tests/ui/test_create_pool_block_pool.py

```diff
@@ -0,0 +1,121 @@
+import logging
+import pytest
+from ocs_ci.framework.pytest_customization.marks import tier1
+from ocs_ci.framework.testlib import skipif_ocs_version, ManageTest, ui
+from ocs_ci.ocs.exceptions import (
+    PoolNotCompressedAsExpected,
+    PoolNotReplicatedAsNeeded,
+    PoolCephValueNotMatch,
+)
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.resources.pod import get_fio_rw_iops
+from ocs_ci.ocs.cluster import (
+    validate_compression,
+    validate_replica_data,
+    check_pool_compression_replica_ceph_level,
+)
+
+logger = logging.getLogger(__name__)
+
+need_to_delete = []
+
+
+@pytest.mark.parametrize(
+    argnames=["replica", "compression"],
+    argvalues=[
+        pytest.param(*[3, True], marks=pytest.mark.polarion_id("OCS-2589")),
+        pytest.param(*[3, False], marks=pytest.mark.polarion_id("OCS-2588")),
+        pytest.param(*[2, True], marks=pytest.mark.polarion_id("OCS-2587")),
+        pytest.param(*[2, False], marks=pytest.mark.polarion_id("OCS-2586")),
+    ],
+)
+class TestPoolUserInterface(ManageTest):
+    """
+    Test Pool User Interface
+
+    """
+
+    pvc_size = 40
+
+    @pytest.fixture()
+    def namespace(self, project_factory):
+        self.proj_obj = project_factory()
+        self.proj = self.proj_obj.namespace
+
+    @pytest.fixture()
+    def storage(self, storageclass_factory_ui, replica, compression):
+        self.sc_obj = storageclass_factory_ui(
+            create_new_pool=True, replica=replica, compression=compression
+        )
+        self.pool_name = self.sc_obj.get()["parameters"]["pool"]
+
+    @pytest.fixture()
+    def pvc(self, pvc_factory):
+        self.pvc_obj = pvc_factory(
+            project=self.proj_obj,
+            interface=constants.CEPHBLOCKPOOL,
+            storageclass=self.sc_obj,
+            size=self.pvc_size,
+        )
+
+    @pytest.fixture()
+    def pod(self, pod_factory):
+        self.pod_obj = pod_factory(pvc=self.pvc_obj)
+
+    @ui
+    @tier1
+    @skipif_ocs_version("<4.8")
```

Comment:
Can you add `@skipif_ui_not_support("block_pool")` decorator?

Reply:
Added